### PR TITLE
Update developer-setup.md

### PR DESCRIPTION
**Pull request by Sayanta66**
Updated link

### CODE REVIEW

**Comment by Sayanta66** [comment]
@cwarnermm and @justinegeffen can you label it?

File: site/content/extend/plugins/developer-setup.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ For developing on Mattermost-managed plugins, each plugin's setup instructions c
 
 **Note:** Deploying with local mode will only work for plugins that have been updated with the functionality from the [Plugin Starter Template](https://github.com/mattermost/mattermost-plugin-starter-template).
 
-If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. Edit your server configuration as follows:
+If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/manage/mmctl-command-line-tool.html) to streamline deploying your plugin. Edit your server configuration as follows:
```
Comment (justinegeffen) [inline site/content/extend/plugins/developer-setup.md:21]: This update is partially correct. The full URL has been updated to include the target section.

Comment (justinegeffen) [inline site/content/extend/plugins/developer-setup.md:21]:
```suggestion
If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/manage/mmctl-command-line-tool.html#local-mode) to streamline deploying your plugin. Edit your server configuration as follows:
```

**Review by justinegeffen — APPROVED** [review]

**Comment by ArmanChand** [comment]
It seems like there is duplicate PR to this :
https://github.com/mattermost/mattermost-developer-documentation/pull/907

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Pull request merged**